### PR TITLE
Fetosense integration

### PR DESCRIPTION
**Pull request by swetatech24**

### CODE REVIEW

**Review by Anjana2310 — APPROVED** [review]
@swetatech24 Looks fine to me

File: src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java [hunk 1]
```diff
@@ -38,13 +38,13 @@
...
 @Table(name = "t_fetosensedata")
-public class Fetosense {
+public class FoetalMonitor {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	@Expose
 	@Column(name = "fetosenseID")
-	private Long partnerFetosenseId;
+	private Long partnerFoetalMonitorId;
```
Comment (harimohanr) [inline src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java:47]: @swetatech24 @Anjana2310 Please change partnerFoetalMonitorId to foetalMonitorId

File: src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java [hunk 2]
```diff
@@ -276,20 +276,20 @@ public void setPartnerName(String partnerName) {
...
 
-	public String getFetosenseMotherID() {
+	public String getFoetalMonitorMotherID() {
 		return cmMotherId;
 	}
 
-	public void setFetosenseMotherID(String fetosenseMotherID) {
-		this.cmMotherId = fetosenseMotherID;
+	public void setFoetalMonitorMotherID(String foetalMonitorMotherID) {
+		this.cmMotherId = foetalMonitorMotherID;
```
Comment (harimohanr) [inline src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java:284]: @swetatech24 @Anjana2310 Can we rename foetalMonitoryMotherID to just montherId?

File: src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java [hunk 3]
```diff
@@ -276,20 +276,20 @@ public void setPartnerName(String partnerName) {
...
 
-	public String getFetosensePartnerID() {
+	public String getFoetalMonitorPartnerID() {
 		return partnerId;
 	}
 
-	public void setFetosensePartnerID(String fetosensePartnerID) {
-		this.partnerId = fetosensePartnerID;
+	public void setFoetalMonitorPartnerID(String foetalMonitorPartnerID) {
+		this.partnerId = foetalMonitorPartnerID;
```
Comment (harimohanr) [inline src/main/java/com/iemr/hwc/data/foetalmonitor/FoetalMonitor.java:292]: @swetatech24 @Anjana2310 Can we rename foetalMonitorPartnerID to just partnerId?

File: src/main/java/com/iemr/hwc/data/login/MasterVan.java [hunk 4]
```diff
@@ -71,35 +71,18 @@ public class MasterVan {
 	@Expose
 	@Column(name = "SwymedEmailID")
 	private String swymedEmailID;
-	
+
 	@Expose
 	@Column(name = "vanfetosenseIDmapped")
-	private Boolean vanfetosenseIDmapped;
-	
+	private Boolean vanfoetalMonitorIDmapped;
```
Comment (harimohanr) [inline src/main/java/com/iemr/hwc/data/login/MasterVan.java:77]: @swetatech24 Shouldn't this field name be vanFoetalMonitorMapped?

**Review by harimohanr — CHANGES_REQUESTED** [review]
@swetatech24 @Anjana2310 Suggested few changes. Please address that and merge it.

**Review by harimohanr — APPROVED** [review]

**Pull request merged**